packages/components/psammead-timestamp/src/index.stories.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import notes from '../README.md';
+import Timestamp from '.';
+
+storiesOf('Timestamp', module)
+  .add(
+    'default',
+    () => <Timestamp datetime="1530947227000">7 July 2018</Timestamp>,
+    { notes },
+  )
+  .add(
+    'with "updated" prefix',
+    () => <Timestamp datetime="1530947227000">Updated 7 July 2018</Timestamp>,
```

Comment:
Could this maybe use a storybook `knob` to vary the text? 

We do have https://www.npmjs.com/package/@bbc/psammead-storybook-helpers but I think it might be overkill for this application, up to you

Reply:
Adding a knob would mean we pull in [TimestampContainer](https://github.com/bbc/simorgh/tree/latest/src/app/containers/Timestamp) from simorgh and we still haven't figured out how to [Split the TimestampContainer logic](https://github.com/bbc/simorgh/issues/1565).

Reply:
I mean just where we have `Updated 7 July 2018` do `text('Timestamp Text', 'Updated 7 July 2018')` from `@storybook/addon-knobs` for example :) Just a simple text box so people can tweak the value if they want to

Reply:
I got it. Done.